### ISSUE TITLE
[DISCUSSION] How do we use the trained data in Clojure

### ISSUE THREAD

**Issue by dhk465** [dhk465/symbolic_comp_one#30]
**Is your feature request related to a problem? Please describe.**
The trained data is in tflite format from Tensorflow. We need to find an appropriate library to process the data in Clojure.

**Describe the solution you'd like**
a library that can both take tflite files and process them to produce similarity maps for a new image

**Describe alternatives you've considered**
Cortex might be able to take Keras models from Tensorflow.

**Comment by dhk465** [comment]
The trained data can be found [here](https://github.com/dhk465/symbolic_comp_one/tree/datasets/trained/tflite).

The final accuracy of the training was 94.65% with 0.14 loss.

**Comment by dhk465** [comment]
https://www.tensorflow.org/guide/saved_model
This tutorial gives an insight on how to use the trained model to predict a new image.

**Comment by ilyassakhanov** [comment]
https://github.com/baldmountain/tf-label-image
Another project, that successfully used TessorFlow+ Clojure 
Edit: No success running it, our team suspects that the code is outdated.

**Comment by dhk465** [comment]
https://androidkt.com/how-to-predict-images-using-trained-keras-model/
https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md
This is another solution to build a model but with our own dataset

**Comment by dhk465** [comment]
https://blog.keras.io/building-powerful-image-classification-models-using-very-little-data.html
This is yet another option to consider.
Since @ilyassakhanov found a library that uses h5 models to label images.

> 
> 
> https://github.com/baldmountain/tf-label-image
> Another project, that successfully used TessorFlow+ Clojure

But it seems to take a significantly more time to train.

**Comment by dhk465** [comment]
https://github.com/dhk465/symbolic_comp_one/tree/datasets/trained
added a keras h5 trained model to the datasets branch

**Comment by ilyassakhanov** [comment]
I will put it here, since i dont know where else to put it.
How to run label-one in mnist example, cortex:
lein run in mnist-classification
(require 'mnist-classification.core)
(in-ns 'mnist-classification.core)
(label-one)

**Comment by dhk465** [comment]
A new branch "classifier-example" contains the example code for our image classification
We are back to Cortex now.
